tgui/packages/tgui/interfaces/ChemMaster.js

```diff
@@ -288,82 +297,239 @@ const ChemMasterProduction = (props, context) => {
 
 const ChemMasterProductionChemical = (props, context) => {
   const { act, data } = useBackend(context);
+  const tabs = [
+    {
+      'name': 'Pills',
+      'icon': 'pills',
+    },
+    {
+      'name': 'Patches',
+      'icon': 'plus-square',
+    },
+    {
+      'name': 'Bottles',
+      'icon': 'wine-bottle',
+    },
+  ];
+  const decideTab = (mode) => {
+    switch (mode) {
+      case 1:
+        return <ChemMasterProductionPills />;
+      case 2:
+        return <ChemMasterProductionPatches />;
+      case 3:
+        return <ChemMasterProductionBottles />;
+      default:
+        return 'UNKNOWN INTERFACE';
+    }
+  };
+  return (
+    <>
+      <Tabs>
+        {tabs.map((t, i) => {
+          i += 1;
```

Comment:
you shouldn't increment i here, `map` increments `i` as the index of its loop automatically